style.css

```diff
@@ -93,3 +118,16 @@ form button {
 .btn {
   float: right;
```

Comment:
- [ ] The button element in the add book form is not displayed as intended which reflects the original design. It is always important to stick with a given design assuming it is a client's requirement. Kindly make changes here! :pray: 

![image](https://user-images.githubusercontent.com/64894927/138346724-d086354f-4ef7-4b1e-83e9-38fc50dbf394.png)
![image](https://user-images.githubusercontent.com/64894927/138346749-d86dae34-8077-4a20-b87d-1e222f373ed1.png)